cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -95,6 +101,31 @@ public void execute(Project project) {
                 start = System.currentTimeMillis();
             }
             jBallerinaBackend.emit(JBallerinaBackend.OutputType.EXEC, executablePath);
+
+            if (enableNativeImage) {
+                // Create nativeConfig directory
+                if (!Files.exists(target.path().resolve("native"))) {
```

Comment:
We can set this to a variable and reuse  in L108, L111
`target.path().resolve("native")`